src/AddExerciseForm.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Button, FormControl, FormGroup, TextField,
+} from '@material-ui/core';
+import {
+  ChangeEvent, useState, FC,
+} from 'react';
+import useLocalStorage from './hooks/useLocalStorage';
+
+const AddExerciseForm: FC = () => {
+  const [formValues, setFormValues] = useState({ description: '', sets: '', reps: '' });
```

Comment:
I think `useState` takes type parameters, In this case it's an `Exercise`.

So I think it can be `const [formValues, setFormValues] = useState<Exercise>(blah);`